cmd/podman/utils.go

```diff
@@ -73,3 +73,12 @@ func checkIfFileExists(name string) bool {
 	}
 	return !file.IsDir()
 }
+
+// Check if a file is or is not a directory
+func fileIsDir(name string) bool {
+	file, err := os.Stat(name)
+	if os.IsNotExist(err) {
+		return false
+	}
+	return file.IsDir()
```

Comment:
If other error, this will blow up since you will be doing
nil.IsDir()

Reply:
Well darn, I'd thought nil wasn't returned there if there was another file error such as perms, but digging through the source at golang, they always return nil. Fixing and thankx.